packages/text-field/index.tsx

```diff
@@ -272,17 +327,14 @@ class TextField extends React.Component {
         {trailingIcon ? this.renderIcon(trailingIcon) : null}
       </div>
     );
-
     if (helperText) {
-      return ([
-        textField, this.renderHelperText(),
-      ]);
+      return [textField, this.renderHelperText()];
```

Comment:
reminder for other PR: change this to Fragment too

Reply:
I changed it